### ISSUE TITLE
新版本的遇到的疑问

### ISSUE THREAD

**Issue by chasingone** [WuJunde/MedSegDiff#103]
作者您好，在老版本的MedSegDiff上面是使用了FF的，为什么你重新修改后改写的UNet中并没有FF变换，这是因为原来的效果不好吗？那么这和论文就不对应了，请问这是什么原因？

**Comment by MurphyYe** [comment]
Yeah, I also have the same problem like u. And the other question is that in the paper said that in attention mechanism it will be used layer norm but I found that actually it is instance norm.

**Comment by goghfan** [comment]
+1 我也发现了上述两个问题，在新版本中默认去除了FFParse，这跟medsegdiff v2的论文流程图对应不起来，想知道是什么原因。

**Comment by cannonli7** [comment]
> +1 我也发现了上述两个问题，在新版本中默认去除了FFParse，这跟medsegdiff v2的论文流程图对应不起来，想知道是什么原因。

你好，我发现v2版本的ssformer好像在代码里没有？v2论文里的ssformer的输入有两个，然后代码里关于qkv的attention模块都是单输入的？

**Comment by SiHengHeHSH** [comment]
I have the same problem.
